src/ConfigGenerator.php

```diff
@@ -15,6 +15,7 @@ class ConfigGenerator
         'logpath' => '\'.__DIR__.\'/log.txt',
         'logfunction' => '',
         'muteLog' => 'false',
+        'configpath' => null
```

Comment:
```suggestion
         'configpath' => __DIR__.'/../configs/'
```

Could we default this value as suggested so the ternary ifs are not required?

Reply:
That's the way I made the first PR, but then I realized, that I need completely random filenames from time to time. We could use a **setFilename** or **setRandomFilename** method or what would be your approach?

Reply:
To avoid breaking changes, I think we should keep the config path defaulted as suggested.

You could also have a `setFilename` method that sets a private member variable, which if set, could be used as an alternate to the `generateFilename()` method.

Reply:
I updated the code, hope you are fine with the changes I've made

Reply:
This looks good.

However, I've just noticed the `delete` method does not actual appear to delete the file?

Reply:
True, sorry for that. I fixed it.

Reply:
Excellent. I will get this merged and release a new version.